kolibri/auth/constants/facility_configuration_presets.json

```diff
@@ -0,0 +1,36 @@
+{
+  "formal": {
+    "name": "Admin-managed",
+    "default": true,
+    "mappings": {
+      "learner_can_edit_username": false,
+      "learner_can_edit_name": false,
+      "learner_can_edit_password": false,
+      "learner_can_sign_up": false,
+      "learner_can_delete_account": false,
+      "learner_can_login_with_no_password": true
```

Comment:
hm this isn't described in the facility set up details. Disregard if you guys discussed this, though